### PR TITLE
Fix voice generator

### PR DESCRIPTION
**Pull request by ryan5453**
Previously the gen_langs.py script was broken, so it wasn't possible to fetch new languages. I wasn't able to find an easy way to get the languages from the JavaScript files, so I decided to just test every Google Translate-supported language and see if it has a voice.

I ran the new script so the langs.py file will be updated.

### CODE REVIEW

**Comment by ryan5453** [comment]
Perhaps the script could be run via a GitHub Action on an interval (not sure what the best interval would be), not sure how you would want to handling bumping the version though.

**Comment by pndurette** [comment]
Oh this is awesome!
The JavaScript route was always very janky and this is much much simpler—thanks!

> Perhaps the script could be run via a GitHub Action on an interval (not sure what the best interval would be), not sure how you would want to handling bumping the version though.

I was actually planning on doing this! As for the version bumping, actually I'm working on revamping the release process and use conventional commits to do automatic version bumps. Which was needed for this to truly work.

If you could actually revert that change to the jump, I'll do it outside of this PR. Thanks!

File: gtts/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "2.2.4"
+__version__ = "2.2.5"
```
Comment (pndurette) [inline gtts/version.py:1]: The version will be bumped outside of this PR

**Review by pndurette — CHANGES_REQUESTED** [review]

**Comment by ryan5453** [comment]
All set.

**Review by pndurette — APPROVED** [review]

**Comment by pndurette** [comment]
> Perhaps the script could be run via a GitHub Action on an interval (not sure what the best interval would be), not sure how you would want to handling bumping the version though.

Just implemented this!
With [scripts/gen_langs.py](scripts/gen_langs.py) to detect changes (and generating a commit message) and a workflow [.github/workflows/update_langs.yml](.github/workflows/update_langs.yml) to create a PR

**Pull request merged**